setup.sql

```diff
@@ -0,0 +1,10 @@
+CREATE DATABASE quill_db WITH 
+    ENCODING = 'UTF8' 
+    TEMPLATE = template0;
+CREATE USER quill_user WITH ENCRYPTED PASSWORD 'PASS_IN_ENV';
+-- Password shouldn't be hard-coded in here either if we are pushing it
+ALTER ROLE quill_user SET client_encoding TO 'utf8';
+ALTER ROLE quill_user SET default_transaction_isolation TO 'read committed';
+ALTER ROLE quill_user SET timezone TO 'UTC';
+GRANT ALL PRIVILEGES ON DATABASE quill_db TO postgres
```

Comment:
should be `quill_user` and not `postgres` since we just created that user with the roles

Reply:
also, could you add `\q` to this file so people know how to quit out of psql command line?

Reply:
Ah okay. Is quill_user like our admin then? OR a stand in for any user? My reasoning is we don't want them to be able to drop tables and stuff. And I'll add the note about `\q`

Reply:
your reasoning makes sense. as the commands are written right now, it's a stand-in for any user. i think we would have to make it a superuser in order for it to be our admin.

Reply:
Ah okay, sounds good!